src/Methods/Chat.php

```diff
@@ -73,9 +71,7 @@ public function kickChatMember(array $params): bool
      */
     public function leaveChat(array $params): bool
     {
-        $this->get('leaveChat', $params);
-
-        return true;
+        return $this->get('leaveChat', $params)->getResult();
```

Comment:
Why is this method not cast to `bool` while the other methods have (See `kickChatmember` & `unbanChatMember`)?

Why is it even being cast, though?